databricks/koalas/frame.py

```diff
@@ -250,24 +250,14 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True, index=Tru
         --------
         to_string : Convert DataFrame to a string.
         """
+        args = locals()
```

Comment:
@rxin, can we add a comment that this should be located at the top always?

Reply:
done